components/Footer/Footer.tsx

```diff
@@ -0,0 +1,18 @@
+import Link from 'next/link';
+
+import styles from './footer.module.scss';
+
+const copyrightYear = new Date().getFullYear();
+
+export const Footer = () => {
+  return (
+    <footer className={styles.footer}>
+      <p className={styles.footerText}>
```

Comment:
Osobne `p` do wrapowania tekstu w footerze, tylko po to by miało jakiś tam margines? Myślę że lepiej zostawić zawartość w samym elemencie footer i dać mu odpowiednie paddingi.

Reply:
@AdamSiekierski zmienione